pkg/parsers/operatingsystem/operatingsystem_linux.go

```diff
@@ -63,14 +61,7 @@ func getValueFromOsRelease(key string) (string, error) {
 		line := scanner.Text()
 		if strings.HasPrefix(line, keyWithTrailingEqual) {
 			data := strings.SplitN(line, "=", 2)
-			values, err := shellwords.Parse(data[1])
-			if err != nil {
-				return "", fmt.Errorf("%s is invalid: %s", key, err.Error())
-			}
-			if len(values) != 1 {
-				return "", fmt.Errorf("%s needs to be enclosed by quotes if they have spaces: %s", key, data[1])
-			}
-			value = values[0]
+			value = strings.Trim(data[1], `"' `) // remove leading/trailing quotes and whitespace
```

Comment:
Taking a slight shortcut here, but I don't expect this to cause problems for most cases. I can make it more strict it this is a concern